decidim-comments/app/frontend/comments/comment.component.tsx

```diff
@@ -157,20 +157,42 @@ class Comment extends React.Component<CommentProps, CommentState> {
     const { comment: { author } } = this.props;
 
     if (author.deleted) {
-      return (
-        <div className="author author--inline">
-          <span className="author__avatar">
-            <img src={author.avatarUrl} alt="author-avatar" />
-          </span>
-          <span className="author__name">
-            <span className="label label--small label--basic">
-              {I18n.t("components.comment.deleted_user")}
-            </span>
-          </span>
-        </div>
-      );
+      return this._renderDeletedAuthor();
     }
 
+    return this._renderActiveAuthor();
+  }
+
+  /**
+   * Render deleted author information
+   * @private
+   * @returns {DOMElement} - Render all the author information
+   */
+  private _renderDeletedAuthor() {
+    const { comment: { author } } = this.props;
+
+    return (
+      <div className="author author--inline">
+        <span className="author__avatar">
+          <img src={author.avatarUrl} alt="author-avatar" />
+        </span>
+        <span className="author__name">
+          <span className="label label--small label--basic">
+            {I18n.t("components.comment.deleted_user")}
+          </span>
+        </span>
+      </div>
+    );
+  }
+
+  /**
+   * Render active author information
+   * @private
+   * @returns {DOMElement} - Render all the author information
+   */
+  private _renderActiveAuthor() {
+    const { comment: { author } } = this.props;
+
```

Comment:
The message of this commit is now incorrect because half of it was included in #2402 (to fix codeclimate there). Worth rebasing?

Reply:
Not doing it, we could edit it right before merge & squash directly.